mailboxes/lang/de

```diff
@@ -77,26 +78,26 @@ mail_tofilter=Filter nach Empfänger erstellen
 mail_subfilter=Filter nach Betreff erstellen
 mail_unknown=Unbekannt
 
-mail_sign=Unterschreibe mit GnuPG-Schlüssel
+mail_sign=Mit GnuPG-Schlüssel unterschreiben
 mail_nosign=&lt;Nicht signieren&gt;
 mail_crypt=GnuPG-Verschlüsselung für:
 mail_nocrypt=&lt;Nicht verschlüsseln&gt;
 mail_samecrypt=&lt;Schlüssel der Zieladressen&gt;
 mail_addresses=Adressbuch bearbeiten
 mail_folders=Ordner bearbeiten
-mail_err=Beim Auflisten der Mails in diesem Ordner ist ein Fehler aufgetreten : $1
+mail_err=Beim Auflisten der Mails in diesem Ordner ist ein Fehler aufgetreten: $1
```

Comment:
Using a space before `:` is something we do everywhere to improve readability. I'd suggest not changing it.